docs/learning_neuroimaging.md

```diff
@@ -0,0 +1,46 @@
+
+# Neuroimaging
+* [Tor Wager's](https://twitter.com/torwager) and Martin Lindquist's 2 parts MOOC on neuroimaging ([part 1](https://en.coursera.org/learn/functional-mri)
+ and [part 2](https://en.coursera.org/learn/functional-mri-2))
+
+
+# Quality control
+Be sure to check the newly formed [Neuroimaging quality control task force](https://crossinvalidation.com/2018/07/31/neuroimaging-quality-control-niqc-task-force-to-develop-protocols-tools-and-manuals/)
+
+* [quality control on MRI and fMRI](https://practicalfmri.blogspot.com/2014/08/qa-for-fmri-part-3-facility-qa-what-to.html)
+* [COBIDacq](https://practicalfmri.blogspot.com/2017/12/cobidacq.html)
+* [In the wikibook neuroimaging data processing](https://en.wikibooks.org/wiki/Neuroimaging_Data_Processing/Data_Quality)
+* [References for quality control](https://www.zotero.org/groups/2221093/niqc)
+
+* [MRIQC](https://mriqc.readthedocs.io/en/stable/) MRI quality control. A BIDS app that runs a pipeline to assess the quality of your data.
+* [the PCP Quality Assessment Protocol](https://github.com/BIDS-Apps/QAP) is another BIDS app based on the protocol of [the connectome project data}(http://preprocessed-connectomes-project.org/quality-assessment-protocol/)
+* [Scripts for quality control of diffusion data](http://davidroalf.com/script_download/)
+* [Qoala-t](https://github.com/Qoala-T/Qoala-T) for QA for freesurfer segmentations also with an online [shinyapp](https://qoala-t.shinyapps.io/qoala-t_app/)
+* [Visual QC](https://github.com/raamana/visualqc) developed by [Pradeep](https://twitter.com/raamana_).
+
+
+# Diffusion Microstructure Imaging in Python (Dmipy)
+The Dmipy software project is dedicated to fasciliting high-level, reproducible diffusion microstructure research.
+* The [Dmipy open-source repository](https://github.com/AthenaEPI/dmipy) with many examples on implementing and fitting microstructure models.
+* Our Preliminary [Reference Paper](https://hal.inria.fr/hal-01873353/file/dmipy-diffusion-microstructure.pdf)
+
+
+# Other specific topics
+* [Computer Vision](https://neurohackademy.org/course/computer-vision/) by Michael Beyeler within Neurohackademy 2018 (53 min)
```

Comment:
I have tried to use the "template" for videos that has the duration as separate "field". Let me know if you think we should use "everywhere".

https://github.com/Remi-Gau/tutorials-and-resources/commit/684fa5cc8af2a73c5fc9ca5bff18c3e6202d37f4

Reply:
Not sure what you mean by "everywhere". Also not sure how to handle date (how specific can/should we be?) Do we have a term that is generic and applies to both summer and winter schools?

```suggestion
* Computer Vision
 - description: Lecture on Computer Vision by Michael Beyeler within Neurohackademy 2018.
 - URL: https://neurohackademy.org/course/computer-vision/
 - date: 2018
 - type: [video] [summer school][image processing][machine learning]
 - duration: 00:53
 - level: [intermediate]
 - event: Neurohackademy 2018
 - by: Michael Beyeler
```

Reply:
By "everywhere" I was thinking that maybe you could give it a try in this page and see what "problems" you encounter. Which is what you did. :-) Sorry I was not clear.

For date I think that we can try to be as specific as the info we have but I would say year is the minimum (given we have the info).

Reply:
Agreed! Let me see what I can do.